app-config/shared/global/dev.conf

```diff
@@ -0,0 +1,250 @@
+commercial.soulmates_url = "param-store@/frontend/dev/commercial.soulmates_url.v1"
+commercial.dfp.guMerchandising.advertiserId = "param-store@/frontend/dev/commercial.dfp.guMerchandising.advertiserId.v1"
+
+# Magento bookshop API properties
+magento.domain = "param-store@/frontend/dev/magento.domain.v1"
+magento.consumer.key = "param-store@/frontend/dev/magento.consumer.key.v1"
+magento.consumer.secret = "param-store@/frontend/dev/magento.consumer.secret.v1"
+magento.access.token.key = "param-store@/frontend/dev/magento.access.token.key.v1"
+magento.access.token.secret = "param-store@/frontend/dev/magento.access.token.secret.v1"
+magento.isbn.lookup.path = "param-store@/frontend/dev/magento.isbn.lookup.path.v1"
+magento.auth.path = "param-store@/frontend/dev/magento.auth.path.v1"
+
+faciatool.show_test_containers = "param-store@/frontend/dev/faciatool.show_test_containers.v1"
+
+#Admin Oauth
+admin.oauth.clientid = "param-store@/frontend/dev/admin.oauth.clientid.v1"
+admin.oauth.callbacks = "param-store@/frontend/dev/admin.oauth.callbacks.v1"
+
+#Preview Oauth
+standalone.oauth.clientid = "param-store@/frontend/dev/standalone.oauth.clientid.v1"
+standalone.oauth.callback = "param-store@/frontend/dev/standalone.oauth.callback.v1"
+
+# Beacon diagnostics
+beacon.url = "param-store@/frontend/dev/beacon.url.v1"
+
+# Headlines AB Test
+headlines.spreadsheet = "param-store@/frontend/dev/headlines.spreadsheet.v1"
+
+# Email signup
+email.signup.url = "param-store@/frontend/dev/email.signup.url.v1"
+
+# Connect straight to production DFP for testing
+guardian.page.dfpAccountId = "param-store@/frontend/dev/guardian.page.dfpAccountId.v1"
+guardian.page.dfpAdUnitRoot = "param-store@/frontend/dev/guardian.page.dfpAdUnitRoot.v1"
+guardian.page.dfp.facebookIaAdUnitRoot = "param-store@/frontend/dev/guardian.page.dfp.facebookIaAdUnitRoot.v1"
+guardian.page.dfp.mobileAppsAdUnitRoot = "param-store@/frontend/dev/guardian.page.dfp.mobileAppsAdUnitRoot.v1"
+
+# News Alert
+news-alert.api.key = "param-store@/frontend/dev/news-alert.api.key.v1"
+
+#HealthCheck refresh interval (0 = no update on regular schedule)
+healthcheck.updateIntervalInSecs = "param-store@/frontend/dev/healthcheck.updateIntervalInSecs.v1"
+
+# r2 Press (empty in dev)
+admin.r2.page.press.sqs.queue.url = "param-store@/frontend/dev/admin.r2.page.press.sqs.queue.url.v1"
+admin.r2.page.press.takedown.sqs.queue.url = "param-store@/frontend/dev/admin.r2.page.press.takedown.sqs.queue.url.v1"
+
+# Properties needed to run Next-Gen
+# these properties should be put inside [HOME_DIRECTORY]/.gu/frontend.properties
+#
+# You should only share this file with other team members, some of the keys in here are not for wider use
+
+############################################################################
+# REQUIRED properties - you need all of these for the project to work
+# you should be fine using the defaults below
+############################################################################
```

Comment:
The 10 lines above are no needed I think

Reply:
👍  This comment was in the S3 config